test/fw/ptl/utils/plugins/ptl_test_runner.py

```diff
@@ -554,11 +555,16 @@ def __get_timeout(self, test):
             method = getattr(test.test, getattr(test.test, '_testMethodName'))
             return getattr(method, TIMEOUT_KEY)
         except:
```

Comment:
you mentioned in the PR description that this block of code should run when there are any attribute errors, so please catch AttributeError specifically in the except statement.

Reply:
@agrawalravi90 In my case, I got AttributeError, but the failure can be anything. That's why I have kept it as general as before.

Reply:
You could be masking errors if you catch all exceptions and treat them as non-fatal, it's a bad coding practice and discouraged by Python's PEP8 standard: https://www.python.org/dev/peps/pep-0008/ (search for "When catching exceptions")

Reply:
@agrawalravi90 "except" in __get_timeout is for the section of the try block, not for any PTL error.  If at all test case fails with any error (IndexError, AttributeError etc), PTL is handiling these and throws proper error as expected. @hirenvadalia Please confirm the same.

Reply:
@shilpakodli I think @agrawalravi90 is right, you only need AttributeError here, have you seen any other errors apart from AttributeError?

Reply:
@hirenvadalia and @agrawalravi90 I have made changes. I agree with you.